### PR TITLE
Add PEG Parser experts

### PR DESCRIPTION
**Pull request by lysnikolaou**

### CODE REVIEW

**Review by gvanrossum — APPROVED** [review]

File: experts.rst [hunk 1]
```diff
@@ -338,6 +339,7 @@ memoryview
 networking          giampaolo.rodola,
 object model        benjamin.peterson, twouters
 packaging           tarek, lemburg, alexis, eric.araujo, dstufft, paul.moore
+peg parser          gvanrossum, pablogsal, lysnikolaou
```
Comment (pablogsal) [inline experts.rst:342]: Maybe PEG instead of "peg"?

**Review by pablogsal — APPROVED** [review]
LGTM

I left a minor comment but feel free to ignore

**Comment by pablogsal** [comment]
> LGTM
> 
> I left a minor comment but feel free to ignore

Oh, I didn't realized that this was merged, my bad (I did the review on the phone and that is not clear from the UI)

**Comment by isidentical** [comment]
Hey @lysnikolaou, today I tried to use the peg parser experts list on an issue, but apparently your bpo name is `lys.nikolaou` on the other hand list contains `lysnikolaou`, FYI.

**Comment by lysnikolaou** [comment]
Thanks for letting me know, @isidentical!

**Pull request merged**